web/src/engine/websites/AllHentai_e2e.ts

```diff
@@ -0,0 +1,26 @@
+import { TestFixture } from '../../../test/WebsitesFixture';
+
+const config = {
+    plugin: {
+        id: 'allhentai',
+        title: 'AllHentai'
+    },
+    container: {
+        url: 'https://20.allhen.online/nochiu_ona_nimfomanka',
+        id: '/nochiu_ona_nimfomanka',
+        title: 'Ночью она нимфоманка'
+    },
+/*   Need to be logged, and chapter link change with user id anyway
+
+    child: {
+        id: '/nochiu_ona_nimfomanka/vol1/4',
```

Comment:
Parameter `?mtr=1` is missing

Reply:
It doesnt matter, as the test will NEVER be good. On this website chapter will ALWAYS has user token as parameter and there is none when not logged. Fine, i'll add it